hc/settings.py

```diff
@@ -145,6 +135,13 @@
 COMPRESS_OFFLINE = True
 
 EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'hcaceskla@gmail.com'
+EMAIL_HOST_PASSWORD = 'hc@ce5kla'
```

Comment:
Configure sensitive information such as the password in the local settings file or the environment. Committing it with your source code could expose a possible vulnerability